Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -4659,6 +4660,9 @@ def return_error(message, error='', outputs=None):
     if not isinstance(message, str):
         message = message.encode('utf8') if hasattr(message, 'encode') else str(message)
 
+    if is_command and demisto.command() == 'get-modified-remote-data' and not isinstance(error, NotImplementedError):
+        message += 'skip update'
```

Comment:
add a space (or a new line) before the word `skip`
in your test, the resulting message is `Test Errorskip update`

Reply:
done